src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/RoundingOps.txt

```diff
@@ -0,0 +1,92 @@
+
+>> Trunc(Sequence(41,-5,0.25),1)
+Table({Value:-5},{Value:-4.7},{Value:-4.5},{Value:-4.2},{Value:-4},{Value:-3.7},{Value:-3.5},{Value:-3.2},{Value:-3},{Value:-2.7},{Value:-2.5},{Value:-2.2},{Value:-2},{Value:-1.7},{Value:-1.5},{Value:-1.2},{Value:-1},{Value:-0.7},{Value:-0.5},{Value:-0.2},{Value:0},{Value:0.2},{Value:0.5},{Value:0.7},{Value:1},{Value:1.2},{Value:1.5},{Value:1.7},{Value:2},{Value:2.2},{Value:2.5},{Value:2.7},{Value:3},{Value:3.2},{Value:3.5},{Value:3.7},{Value:4},{Value:4.2},{Value:4.5},{Value:4.7},{Value:5})
+
+// 2 digits right
+
+>> Round(Sequence(41,-5,0.25),2)
+Table({Value:-5},{Value:-4.75},{Value:-4.5},{Value:-4.25},{Value:-4},{Value:-3.75},{Value:-3.5},{Value:-3.25},{Value:-3},{Value:-2.75},{Value:-2.5},{Value:-2.25},{Value:-2},{Value:-1.75},{Value:-1.5},{Value:-1.25},{Value:-1},{Value:-0.75},{Value:-0.5},{Value:-0.25},{Value:0},{Value:0.25},{Value:0.5},{Value:0.75},{Value:1},{Value:1.25},{Value:1.5},{Value:1.75},{Value:2},{Value:2.25},{Value:2.5},{Value:2.75},{Value:3},{Value:3.25},{Value:3.5},{Value:3.75},{Value:4},{Value:4.25},{Value:4.5},{Value:4.75},{Value:5})
+
+>> RoundUp(Sequence(41,-5,0.25),2)
+Table({Value:-5},{Value:-4.75},{Value:-4.5},{Value:-4.25},{Value:-4},{Value:-3.75},{Value:-3.5},{Value:-3.25},{Value:-3},{Value:-2.75},{Value:-2.5},{Value:-2.25},{Value:-2},{Value:-1.75},{Value:-1.5},{Value:-1.25},{Value:-1},{Value:-0.75},{Value:-0.5},{Value:-0.25},{Value:0},{Value:0.25},{Value:0.5},{Value:0.75},{Value:1},{Value:1.25},{Value:1.5},{Value:1.75},{Value:2},{Value:2.25},{Value:2.5},{Value:2.75},{Value:3},{Value:3.25},{Value:3.5},{Value:3.75},{Value:4},{Value:4.25},{Value:4.5},{Value:4.75},{Value:5})
+
+>> RoundDown(Sequence(41,-5,0.25),2)
+Table({Value:-5},{Value:-4.75},{Value:-4.5},{Value:-4.25},{Value:-4},{Value:-3.75},{Value:-3.5},{Value:-3.25},{Value:-3},{Value:-2.75},{Value:-2.5},{Value:-2.25},{Value:-2},{Value:-1.75},{Value:-1.5},{Value:-1.25},{Value:-1},{Value:-0.75},{Value:-0.5},{Value:-0.25},{Value:0},{Value:0.25},{Value:0.5},{Value:0.75},{Value:1},{Value:1.25},{Value:1.5},{Value:1.75},{Value:2},{Value:2.25},{Value:2.5},{Value:2.75},{Value:3},{Value:3.25},{Value:3.5},{Value:3.75},{Value:4},{Value:4.25},{Value:4.5},{Value:4.75},{Value:5})
+
+>> Trunc(Sequence(41,-5,0.25),2)
+Table({Value:-5},{Value:-4.75},{Value:-4.5},{Value:-4.25},{Value:-4},{Value:-3.75},{Value:-3.5},{Value:-3.25},{Value:-3},{Value:-2.75},{Value:-2.5},{Value:-2.25},{Value:-2},{Value:-1.75},{Value:-1.5},{Value:-1.25},{Value:-1},{Value:-0.75},{Value:-0.5},{Value:-0.25},{Value:0},{Value:0.25},{Value:0.5},{Value:0.75},{Value:1},{Value:1.25},{Value:1.5},{Value:1.75},{Value:2},{Value:2.25},{Value:2.5},{Value:2.75},{Value:3},{Value:3.25},{Value:3.5},{Value:3.75},{Value:4},{Value:4.25},{Value:4.5},{Value:4.75},{Value:5})
+
+>> Sum(ForAll(Sequence(41,-5,0.25),Round(Value,2)=RoundDown(Value,2) && Round(Value,2)=RoundUp(Value,2) && Round(Value,2)=Trunc(Value,2)),Value)
+41
+
+// *********** Big numbers
```

Comment:
Agreed.  But since float is messed up for big numbers, we should wait for that to get fixed first.  https://github.com/microsoft/Power-Fx/issues/1331.  I made a note in that issue that we should add tests here too.